# Goal
float64 x
float64 y
---
# Result
string result_message
---
# Feedback
float64 current_x
float64 current_y

